profiles/base.yaml

```diff
@@ -504,6 +504,25 @@ workorder_max_items: 3
 workorder_recipes:
 workorder_cash_on_hand:
 
+#CARAVAN SETTINGS
+# container holding caravan contracts for ;trade
+trade_contract_container: backpack
+# number of copper to carry on-hand while running ;trade
+caravan_coins_on_hand: 9555
+# Have you purchased a 750plat interior from the trade minister?
+caravan_interior: false
+# skills and cooldowns to train in ;trade.  See https://github.com/rpherbig/dr-scripts/wiki/Trader-Tutorials for valid skills and what they do
+caravan_training_skills:
```

Comment:
Noticed trade does first aid training, should this be included here?

Reply:
As below, there's now a check in there, so adding it in shouldn't hurt anyone. Before this goes live I'll do a full review of base skills and add anything in that has an escape clause.